testsuite/tests/typing-modules/aliases.ml

```diff
@@ -661,7 +661,7 @@ module rec Bad : A = Bad;;
 [%%expect{|
 module type Alias = sig module N : sig  end module M = N end
 module F : functor (X : sig  end) -> sig type t end
-Line 1:
+Characters -1--1:
```

Comment:
This looks to me like a regression.

Reply:
I would argue it is not. The `Line 1` here was nonsensical and unrelated to the actual location of the error; it is the result of a dummy location leaking out. A dummy location is now printed as `Characters -1--1`, which I think is slightly better because it makes it clear that it is a dummy location.
I don't think it is worth trying to improve this situation printing time: the proper fix should instead be to not emit dummy locations in places where actual locations are expected.

Reply:
OK, I agree.